acquisitions/src/main/resources/thunderjet/mod-orders/orders-junit.feature

```diff
@@ -39,20 +39,21 @@ Feature: mod-orders integration tests
 
 
     * table userPermissions
-      | name                                   |
-      | 'orders.all'                           |
-      | 'finance.all'                          |
-      | 'inventory.all'                        |
-      | 'tags.all'                             |
-      | 'orders.item.approve'                  |
-      | 'orders.item.reopen'                   |
-      | 'orders.item.unopen'                   |
-      | 'invoice.all'                          |
-      | 'audit.all'                            |
-      | 'orders-storage.claiming.process'      |
-      | 'inventory-storage.instances.item.get' |
-      | 'inventory-storage.items.item.get'     |
-      | 'orders-storage.titles.item.get'       |
+      | name                                           |
+      | 'orders.all'                                   |
+      | 'finance.all'                                  |
+      | 'inventory.all'                                |
+      | 'tags.all'                                     |
+      | 'orders.item.approve'                          |
+      | 'orders.item.reopen'                           |
+      | 'orders.item.unopen'                           |
+      | 'invoice.all'                                  |
+      | 'audit.all'                                    |
+      | 'orders-storage.claiming.process'              |
+      | 'inventory-storage.instances.item.get'         |
+      | 'inventory-storage.items.item.get'             |
+      | 'orders-storage.titles.item.get'               |
+      | 'orders-storage.po-lines-batch.collection.put' |
```

Comment:
I wonder maybe this permission is not need. these permission usually need when calling individual api  for example  trying to get individual item in scenarios to verify details let's say, so we need | 'inventory-storage.items.item.get'     |.

The case we don't verify item details, but api will use this fetch inventory items api inside of process then, we don't need the permission to fetch items because it is already exists in ModuleDescriptor for individual api under modulePermission array. 

As a conclusion, this permission maynot need unless we use the permission as scenario.

Reply:
The Karate tests do work without it, plus it is explicitly used by in the new associated PR: 
![image](https://github.com/user-attachments/assets/a0b72340-75d4-4b1d-9291-7c2802eb310e)